### DISCUSSION TITLE
Working with old R projects.

### DISCUSSION THREAD

**Discussion by SETOsfk** [posit-dev/positron#3796] [Q&A] [answered]
Is there any way to work old R projects or should i just use r scripts.

**Comment by juliasilge** [comment, marked as answer]
You absolutely can work with RStudio projects, yes! There are two ways I like to open them up:

- Use "Open Folder" (from the workspace switcher on the upper right, native app toolbar, or welcome page) to open the RStudio project folder. This makes the folder your Positron workspace.
- From the terminal, `cd` to the RStudio project folder and then do `positron .` in the same way that you might have done `open my_project.Rproj` for RStudio. This also make the RStudio project your current Positron workspace. To make this work, you first need to run this command one time to add Positron to your path:

![Screenshot 2024-07-02 at 9 23 54 AM](https://github.com/posit-dev/positron/assets/12505835/67f658a4-c72e-4f63-a4c9-53b154beac02)
